eq-author/src/components/CommentsPanel/index.js

```diff
@@ -398,7 +390,11 @@ const CommentsPanel = ({ componentId, me: { id: myId } }) => {
 
   const displayComments = comments.map((item, index) => {
     const replies = comments[index].replies;
```

Comment:
```suggestion
    const { replies } = comments[index];
```

Whilst you're here, you can modernise the style of this line to use destructuring.